src/main/kotlin/blackjack/domain/player/Hand.kt

```diff
@@ -0,0 +1,32 @@
+package blackjack.domain.player
+
+import blackjack.domain.deck.Card
+
+class Hand {
+    var status: HandStatus = HandStatus.GENERAL
+        private set
+
+    private val _cards: ArrayList<Card> = arrayListOf()
+
+    val cards: List<Card>
+        get() = _cards
```

Comment:
```suggestion
        get() = _cards.toList()
```

실제 내부 구현은 Mutable하므로, 내보낼 때 toList 등으로 Immutable 하게 변환해주면 어떨까요?
`List` 반환 타입이더라도 형 변환을 통해 값을 추가하거나 할 수 있어요.